packages/@webex/contact-center/src/config.ts

```diff
@@ -0,0 +1,16 @@
+export default {
+  cc: {
+    allowMultiLogin: true,
+    clientType: 'WebexCCSDK',
+    isKeepAliveEnabled: false,
+    force: true,
+    metrics: {
+      clientName: 'WEBEX_JS_SDK',
+      clientType: 'WebexCCSDK',
+    },
+    logging: {
```

Comment:
are we not going to follow a similar logging object like calling or other plugins?

Reply:
We will, it was just for internal config use I put it for now. Removed it